tutorials/scripting/cross_language_scripting.rst

```diff
@@ -192,7 +192,7 @@ to said method.
 
     // When dealing with functions taking a single array as arguments, we need to be careful.
     // If we don't cast it into an object, the engine will treat each element of the array as a separate argument and the call will fail.
-    String[] arr = new String[] { "a", "b", "c" };
+    string[] arr = new string[] { "a", "b", "c" };
     // myGDScriptNode.Call("print_array", arr); // This line will fail silently and won't error out.
     myGDScriptNode.Call("print_array", (object)arr); // a, b, c
     myGDScriptNode.Call("print_array", (object)new int[] { 1, 2, 3 }); // 1, 2, 3
```

Comment:
These would have to be `Variant` in 4.0 instead of `System.Object`. I think arrays are converted implicitly to `Variant` now, so removing the cast should work.

```suggestion
    myGDScriptNode.Call("print_array", arr); // a, b, c
    myGDScriptNode.Call("print_array", new int[] { 1, 2, 3 }); // 1, 2, 3
```

Reply:
So, you're right on this one. I can update this. But a lot of this page is now outdated.  
It's currently not possible to set C# fields from GDScript (line 127), get or set C# properties from GDScript (line 130), etc.